etl/search.py

```diff
@@ -8,8 +8,14 @@
     open(Path(__file__).resolve().parent / "metrics_search.js.tmpl").read()
 )
 
+FOG_AND_LEGACY_SEARCH_JS_TEMPLATE = jinja2.Template(
+    open(Path(__file__).resolve().parent / "fog_and_legacy.js.tmpl").read()
+)
+
+FOG_DATA_JS_TEMPLATE = jinja2.Template(open(Path(__file__).resolve().parent / "fog.js.tmpl").read())
+
 
-def create_metrics_search_js(metrics, legacy=False):
+def create_metrics_search_js(metrics, app_name=None, legacy=False):
```

Comment:
Can we add a docstring here explaining what this function does and calling out that "fog" and "fog_an_legacy" are special values and will result in different behaviour of the function? 

Another option would be to create separate functions for the special cases instead of having the special value pattern. (And add docstrings for each of them).